packages/docs/src/page-configs/getting-started/kitchensink/page-config.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiDocsBlock, BlockType } from '@/types/configTypes'
+import { shallowReactive } from 'vue'
+
+const importedConfigsContext = require.context(
+  '@/page-configs/ui-elements/',
+  true,
+  /page-config.ts$/,
+  'lazy',
+)
+
+export const config = shallowReactive<ApiDocsBlock[]>([])
+
+importedConfigsContext.keys().forEach(async (filename) => {
+  let importedConfig = null
+  try {
+    importedConfig = (await importedConfigsContext(filename)).default
+  } catch {
+    return
+  }
+
+  config.push({ ...importedConfig[0], type: BlockType.SUBTITLE }) // title
```

Comment:
> importedConfig[0]

Maybe you can instead find title, it's not immediately clear what you're trying to do.

Reply:
Yep, looking for title block now